packages/ckeditor5-table/src/tableconfig.ts

```diff
@@ -217,6 +217,7 @@ export interface TablePropertiesConfig {
 	borderColors?: TableColorConfig;
 	backgroundColors?: TableColorConfig;
 	defaultProperties?: TablePropertiesOptions;
+	colorPicker?: false | ColorPickerConfig;
```

Comment:
I think we should mention this config somewhere in the doclets of `TableConfig` (`tableProperties?: TablePropertiesConfig;`)